stuffer/s2n_stuffer_text.c

```diff
@@ -78,22 +78,24 @@ int s2n_stuffer_read_expected_str(struct s2n_stuffer *stuffer, const char *expec
 /* Read from stuffer until the target string is found, or until there is no more data. */
 int s2n_stuffer_skip_read_until(struct s2n_stuffer *stuffer, const char *target)
 {
+    PRECONDITION_POSIX(s2n_stuffer_is_valid(stuffer));
+    notnull_check(target);
     int len = strlen(target);
     while (s2n_stuffer_data_available(stuffer) >= len) {
```

Comment:
This whole algorithm seems ... quite slow.  Not worth fixing here, but make an issue to fix it.

Reply:
Tracking https://github.com/awslabs/s2n/issues/2125